cheri_compressed_cap_common.h

```diff
@@ -337,6 +337,18 @@ static inline bool _cc_N(bounds_bits_valid)(_cc_bounds_bits bounds) {
     }
 }
 
+#ifdef CC_IS_MORELLO
+/// Returns the address with Morello flags (high address bits) removed and sign extended.
+static inline _cc_addr_t _cc_N(morello_cap_bounds_address)(_cc_addr_t addr) {
```

Comment:
The morello_ feels redundant when it'll be cc128m_...?

Reply:
True, I could just use that instead of the _cc_N macro

Reply:
Dropped the commit instead.